src/vm/i386/excepx86.cpp

```diff
@@ -38,6 +38,7 @@
 #include "asmconstants.h"
 #include "virtualcallstub.h"
 
+#ifndef WIN64EXCEPTIONS
```

Comment:
Hmm, CLRNoCatchHandler is really needed, I have not realized it is defined here. And maybe I am wrong with excluding the whole file. Let me go through it and see what is actually needed.